src/commands/sourcemaps/upload.ts

```diff
@@ -245,7 +245,7 @@ export class UploadCommand extends Command {
 
   private getRequestBuilder(): RequestBuilder {
     if (!this.config.apiKey) {
-      throw new InvalidConfigurationError(`Missing ${chalk.bold('DATADOG_API_KEY')} in your environment.`)
+      throw new InvalidConfigurationError(`Missing ${chalk.bold('DD_API_KEY')} in your environment.`)
```

Comment:
Let's keep mentioning both in error messages, as it's the original behavior
```suggestion
      throw new InvalidConfigurationError(`Missing ${chalk.bold('DATADOG_API_KEY')} or ${chalk.bold('DD_API_KEY')} in your environment.`)
```